education.html

```diff
@@ -6,28 +6,44 @@
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <title>Education</title>
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" />  
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />  
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" /> 
```

Comment:
Don't remove and add the same line again please

Reply:
oh sorry I didn't mean to.